src/utils/hooks/useCreateReview.tsx

```diff
@@ -40,7 +46,24 @@ const useCreateReview = ({ reviewerId, serialNumber, onCompleted }: UseCreateRev
     ],
   })
 
-  const createReview = ({ reviewAssigmentId, applicationResponses }: CreateReviewProps) => {
+  const createReviewFromStructure = async (structure: FullStructure) => {
+    const elements = Object.values(structure?.elementsById || {})
+    const reviewableElements = elements.filter((element) => element?.isAssigned)
+
+    const applicationResponses = reviewableElements.map((element) => ({
+      applicationResponseId: element.response?.id || 0,
+      reviewQuestionAssignmentId: element.assignmentId,
+    }))
+
+    return await createReviewMutation({
+      variables: {
+        reviewAssigmentId,
+        applicationResponses,
+      },
+    })
+  }
+
+  const createReview = ({ applicationResponses }: CreateReviewProps) => {
```

Comment:
This is here only for existing use? Good to leave a comment for this to be removed. I suggest:
`// TODO: Remove this`

If you do a global search there are many places with this comment to help us when deleting all the old stuff

Reply:
cd40cf8f3145b29aa37e8877578b52d05bd361d8